english/beach/beach.md

```diff
@@ -0,0 +1,8 @@
+You know in your heart the beach is where you wanna be.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.